node/bft/src/primary.rs

```diff
@@ -71,6 +71,8 @@ use tokio::{
     task::JoinHandle,
 };
 
+const MAX_OUTSTANDING_CERTIFICATE_REQUESTS: u32 = 20;
```

Comment:
Might be good to add a comment that indicates why we picked 20? Also maybe write that this is per peer and not the total number of outstanding requests (if I understand correctly).